addons/api/tests/unit/serializers/storage-bucket-test.js

```diff
@@ -125,13 +123,8 @@ module('Unit | Serializer | storage bucket', function (hooks) {
           region: 'eu-west-1',
           access_key_id: 'foobars',
           secret_access_key: 'testing',
-          secrets_hmac: 'hmac',
           disable_credential_rotation: true,
           version: 1,
-          role_arn: null,
-          role_external_id: null,
-          role_session_name: null,
-          role_tags: [],
```

Comment:
Can we also set values for these to confirm the serialization correctly sets them to null?